### PR TITLE
[Feat]:Cart Management Functionality added

### PR DESCRIPTION
**Pull request by t007rushi**
Cart Management Functionality:
✅ADD to cart
✅ REMOVE from cart
✅ Increase the quantity⬆
✅ Decrease the quantity⬇

recording

<details><summary>Browser metadata</summary>

```
Path:      /cart
Browser:   Chrome 99.0.4844.74 on Windows 10
Viewport:  1536 x 775 @1.25x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-7--game-kart-ecom.netlify.app%2Fcart&os=Windows&os_version=10&browser=Chrome&browser_version=99.0&resolution=1536x775&speed=1&start=true&ref=netlify-source)

</details>

[Open Deploy Preview](https://deploy-preview-7--game-kart-ecom.netlify.app/cart) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=62385e0921ef4500084a8539&commentID=62385f7573666719b52975ae&resolution=resolved)

### CODE REVIEW

File: src/components/Card/Card.jsx [hunk 1]
```diff
@@ -33,23 +39,31 @@ const Card = ({prod}) => {
 
           {/* button2 */}
 
-          <button className="card-btn card-icon-btn" onClick={() => {}}>
-            <span className="material-icons icon">shopping_cart</span>
-            <p>Add to Cart</p>
-          </button>
+          {cartArr.find((cartItem) => cartItem.id === prod.id) ? (
```
Comment (JantuDeb) [inline src/components/Card/Card.jsx:42]: Use  `cartArr.some()` if only boolean value is needed

File: src/components/Card/Card.jsx [hunk 2]
```diff
@@ -33,23 +39,31 @@ const Card = ({prod}) => {
...
+            <Link style={{ textDecoration: "none" }} to="/cart">
```
Comment (JantuDeb) [inline src/components/Card/Card.jsx:43]: You can use a `className`  instead of `inline style`

**Review by JantuDeb — APPROVED** [review]
Codes look fine, you can go through the comments I have added.

**Review by ShreyashMahajan — APPROVED** [review]
code looks fine to merge.

**Pull request closed without merging**